spacy/tests/regression/test_issue4104.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+import pytest
+from ..util import get_doc
+
+@pytest.mark.parametrize('text', ['dry spun spun-dry'])
+
+def test_issue4104(en_tokenizer, en_vocab, text):
+    """Test that English lookup lemmatization of spun & dry are correct"""
+    doc = get_doc(en_vocab, [t for t in text.split(" ")])
+    expected = {'dry': 'dry', 'spun': 'spin', 'spun-dry': 'spin-dry'}
+    assert [token.lemma_ for token in doc] == list(expected.values())
```

Comment:
Thanks for adding the test! Looks like this one failed on Python 3.5 because dicts aren't ordered yet, so `values()` returns the key in a different order. (Totally not your fault btw, it's not exactly intuitive.) Calling `sorted` around both lists should resolve this and make sure the order is always the same.

Reply:
Thanks. I went ahead and streamlined the test to not parametrize the string and compare it directly to a list of expected results.